todoapp/app/src/androidTestMock/java/com/example/android/architecture/blueprints/todoapp/addedittask/AddEditTaskScreenTest.java

```diff
@@ -73,16 +75,15 @@ public void registerIdlingResource() {
 
     @Test
     public void errorShownOnEmptyTask() {
-        // Add task title and description
+        // Add invalid title and description combination
```

Comment:
you don't need to do any type or close the keyboard to show an empty task. the click on the fab button can be performed directly.